github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -138,6 +138,46 @@ presubmits:
             memory: "4Gi"
         securityContext:
           runAsUser: 0
+  - always_run: false
+    cluster: kubevirt-prow-control-plane
+    decorate: true
+    name: pull-project-infra-lint
+    run_if_changed: "external-plugins/.*|releng/.*|robots/.*"
+    spec:
+      containers:
+      - args:
+        - make lint
+        command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-ce"
+        image: quay.io/kubevirtci/golang:v20241206-ae344ab
+        resources:
+          requests:
+            memory: "1Gi"
+          limits:
+            memory: "1Gi"
+  - always_run: false
+    cluster: kubevirt-prow-control-plane
+    decorate: true
+    name: pull-project-infra-coverage
+    run_if_changed: "external-plugins/.*|releng/.*|robots/.*"
+    spec:
+      containers:
+      - args:
+        - |
+          GOBIN=/usr/local/bin/ go install github.com/onsi/ginkgo/v2/ginkgo@v2.9.2
```

Comment:
Having a specific image for this would remove the need for this.

Reply:
true - but since we only need this in two cases currently I am not sure whether it's worth it. Also I don't want to put this into the golang image, doesn't seem like a good fit. Maybe some project-infra-test image? WDYT?
(I am not really sure whether we would want to tackle that in this PR, maybe create an issue as tracker?)